test/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
 		"parserOptions": {
-				"ecmaVersion": 6,
+				"ecmaVersion": 2017,
```

Comment:
Why is this needed?

Reply:
Because I needed to use async/await in the tests to wait for the `setState` callback to complete.